### ISSUE TITLE
Set and Get `CA\AuditFilter`

### ISSUE THREAD

**Issue by Viajaz** [PKISolutions/PSPKI#172]
Being able to read and set an ADCS instances `AuditFilter` value would be useful.
https://www.pkisolutions.com/enabling-active-directory-certificate-services-adcs-advanced-audit/

I'm sorry if this feature already exists, I couldn't find it in the documentation nor code.

**Comment by Viajaz** [comment]
https://www.pkisolutions.com/tools/pspki/Set-OnlineResponderProperty/
The `-AuditFlag` parameter

**Comment by Crypt32** [comment]
> https://www.pkisolutions.com/tools/pspki/Set-OnlineResponderProperty/ The `-AuditFlag` parameter

this parameter enables only OCSP audit. This doesn't affect CA audit setting.

**Comment by Viajaz** [comment]
You're absolutely right, no idea how I missed that when it's staring me in the face.
